lte/protos/session_manager.proto

```diff
@@ -792,33 +816,54 @@ enum SMSessionFSMState {
 //PDU session Establishment from AMF to SMF
 //SET MESSSAGE
 message M5GSMSessionContext {
-  bytes pdu_session_id = 1;   //One byte
-  RequestType rquest_type = 2;        //Define ENUM 24-501 - 9.11.3.47 . 1 byte. i.e. initial requerst, existing pdusession etc...
-  RedirectServer pdu_address = 3; //IP address. Imported from session_manager.proto file
+  uint32 pdu_session_id = 1;
+ //Define ENUM 24-501 - 9.11.3.47 . 1 byte. i.e. initial requerst, existing pdusession etc...
+  RequestType rquest_type = 2;
+  //IP address. Imported from session_manager.proto file
+  RedirectServer pdu_address = 3;
   priorityaccess priority_access = 4;
-  AccessType access_type = 5;     //Define enum TS-29.571 - section 5.4.3
-  string imei = 8; // pei - permanent equipment ID. Variable length of 16 bytes. TS-23.003 - 6.4 Like IMEI or IMEISV
-  string gpsi = 9; //Lenght is 10 bytes + 2 bytes country code generic public subscription id.
-  string user_location_info = 10; //up to 6 byte. TS-23.003 - 28.15.3
-  string pcf_id = 11; //length 16 bytes as UUID "Universally Unique IDentifier" TS-29.571 to ietf rfc 4122
+  //Define enum TS-29.571 - section 5.4.3
+  AccessType access_type = 5;
+  // pei - permanent equipment ID. Variable length of 16 bytes. TS-23.003 - 6.4 Like IMEI or IMEISV
+  string imei = 6;
+  //Lenght is 10 bytes + 2 bytes country code generic public subscription id.
+  string gpsi = 7; 
+  //up to 6 byte. TS-23.003 - 28.15.3
+  string user_location_info = 8;
+  //length 16 bytes as UUID "Universally Unique IDentifier" TS-29.571 to ietf rfc 4122
+  string pcf_id = 9;
   //Variable length of 20 bytes. TS-23.003 - 28.8 & TS-23.501 5.9.8
-  repeated LadnServiceArea ladn_service_area = 12; //Length is 10 bytes, like PLMN
-  bool pdu_session_status_notification_sub = 13;   //Subscription For PDU Session Status Notification true or false
-  DnnSelectionMode dnn_selection_mode = 14;       // TS-29-502 - 6.1.6.3.10
-  string trace_requirment = 15;                   // TS-29.571 5.6.
-  bytes integrity_protection_max_data_rate = 16;  //Length is 3 bytes, TS-24.501 9.11.4.7
-  PduSessionType pdu_session_type = 17;
-  SscMode ssc_mode = 18;
-  M5GSMCapability m5g_sm_capability = 19;  // max size go up to 15 bytes. TLV and 9.11.4.1.
-  uint32 max_pkt_filter_supported = 20;    //Maximum no. of supported packet filters
-  bool pdu_session_req_always_on = 21;     //Always-on PDU session requested TS-24.501 9.11.4.4
-  string sm_pdu_dn_req = 22;               //SM PDU DN request container. Max size up to 255. TLV and 9.11.4.16.
-  M5GSMCause m5gsm_cause = 23;
-  QosRules requested_qos_rules = 24; //TS-24501 - 9.11.4.13 minimum 7 and max 65538, variable length.
+  repeated LadnServiceArea ladn_service_area = 10; 
+  //Subscription For PDU Session Status Notification true or false
+  bool pdu_session_status_notification_sub = 11;
+  // TS-29-502 - 6.1.6.3.10 
+  DnnSelectionMode dnn_selection_mode = 12;
+  // TS-29.571 5.6.
+  string trace_requirment = 13;
```

Comment:
"requirment" is a misspelling of "requirement"

Reply:
Fixed it.